medusa/indexers/indexer_config.py

```diff
@@ -1,12 +1,14 @@
 # coding=utf-8
 
+"""Indexer config module."""
+
 import re
 
 from .tmdb.tmdb import Tmdb
 from .tvdbv2.tvdbv2_api import TVDBv2
 from .tvmaze.tvmaze_api import TVmaze
 from ..app import BASE_PYMEDUSA_URL
-from ..session.core import MedusaSession
+from ..session.core import MedusaSafeSession
```

Comment:
I'd rather keep this MedusaSession as long as possible. Any reason you see to change this?
I know we do some exception handling on this. So this could potentially mess that up.